File: examples/cache/set/go/setAValue.go
package example

import (
	"fmt"
	"github.com/micro/services/clients/go/cache"
	"os"
)

// Set an item in the cache. Overwrites any existing value already set.
func SetAvalue() {
	cacheService := cache.NewCacheService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cacheService.Set(&cache.SetRequest{
		Key:   "foo",
		Value: "bar",
	})
	fmt.Println(rsp, err)
}

File: examples/cache/set/node/setAValue.js
const { CacheService } = require("m3o/cache");

// Set an item in the cache. Overwrites any existing value already set.
async function setAvalue() {
  let cacheService = new CacheService(process.env.MICRO_API_TOKEN);
  let rsp = await cacheService.set({
    key: "foo",
    value: "bar",
  });
  console.log(rsp);
}

setAvalue();
